code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -423,14 +426,14 @@ There are several things that need to be remembered:
 		var/obj/item/worn_item = head
 		var/mutable_appearance/head_overlay
 		update_hud_head(worn_item)
-		var/handled_by_bodytype = FALSE
+		var/handled_by_bodytype = TRUE
```

Comment:
?

Reply:
They're supposed to be TRUE by default. It being FALSE in this proc was a typo as far as I can tell, as that would render the var useless.